Generated unit test for `google-apps-chat` fails with `ValueError: Protocol message Icon has no "material_icon" field.`

See build log [here](https://github.com/googleapis/google-cloud-python/actions/runs/8696663810/job/23850434391?pr=12574) from PR https://github.com/googleapis/google-cloud-python/pull/12574

```
>           response = client.update_message(request)

tests/unit/gapic/chat_v1/test_chat_service.py:10332: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
google/apps/chat_v1/services/chat_service/client.py:1564: in update_message
    request = gc_message.UpdateMessageRequest(request)
.nox/unit-3-8/lib/python3.8/site-packages/proto/message.py:609: in __init__
    pb_value = marshal.to_proto(pb_type, value)
.nox/unit-3-8/lib/python3.8/site-packages/proto/marshal/marshal.py:228: in to_proto
    pb_value = self.get_rule(proto_type=proto_type).to_proto(value)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <proto.marshal.rules.message.MessageRule object at 0x7f07f5713e50>
value = {'accessory_widgets': [{'button_list': {}}], 'action_response': {'dialog_action': {'action_status': {'status_code': 1,...nd_name_value', 'triggers_dialog': True, ...}, 'start_index': 1189, ...}], 'argument_text': 'argument_text_value', ...}

    def to_proto(self, value):
        if isinstance(value, self._wrapper):
            return self._wrapper.pb(value)
        if isinstance(value, dict) and not self.is_map:
            # We need to use the wrapper's marshaling to handle
            # potentially problematic nested messages.
            try:
                # Try the fast path first.
>               return self._descriptor(**value)
E               ValueError: Protocol message Icon has no "material_icon" field.

.nox/unit-3-8/lib/python3.8/site-packages/proto/marshal/rules/message.py:36: ValueError
```

--- a/gapic/templates/_pypi_packages.j2
+++ b/gapic/templates/_pypi_packages.j2
@@ -7,7 +7,7 @@ allowed version.
 Note: Set the minimum version for google-cloud-documentai to 2.0.0 which has support for `barcode` in `google.cloud.documentai.types`
 -->
 {% set pypi_packages = {
-    ("google", "apps", "card", "v1"): {"package_name": "google-apps-card", "lower_bound": "0.1.0", "upper_bound": "1.0.0dev"},
+    ("google", "apps", "card", "v1"): {"package_name": "google-apps-card", "lower_bound": "0.1.2", "upper_bound": "1.0.0dev"},
     ("google", "apps", "script", "type"): {"package_name": "google-apps-script-type", "lower_bound": "0.2.0", "upper_bound": "1.0.0dev"},
     ("google", "geo", "type"): {"package_name": "google-geo-type", "lower_bound": "0.1.0", "upper_bound": "1.0.0dev"},
     ("google", "identity", "accesscontextmanager", "v1"): {"package_name": "google-cloud-access-context-manager", "lower_bound": "0.1.2", "upper_bound": "1.0.0dev"},

